src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -16,13 +18,27 @@ import VenueLandingPageContent from "./VenueLandingPageContent";
 import "./VenueLandingPage.scss";
 
 export const VenueLandingPage: React.FC = () => {
-  useConnectCurrentVenue();
   const venueId = useVenueId() || "";
 
   const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
   const venueRequestStatus = useSelector(
     (state) => state.firestore.status.requested.currentVenue
   );
+  const { descendantVenues, sovereignVenueId } = useRelatedVenues({
+    currentVenueId: venueId,
+  });
+
+  const venueEvents = useSelector(venueEventsSelector);
+  const allVenueEvents = useMemo(
+    () => [
+      ...(venueEvents || []),
+      ...(sovereignVenueId === venueId
+        ? [...descendantVenues.flatMap((venue) => venue.events || [])]
+        : []),
+    ],
+    [descendantVenues, venueEvents, sovereignVenueId, venueId]
+  );
```

Comment:
I believe you should use `useVenueEvents({
    venueIds: [currentVenueId, ...descedantVenueIds]
  })` here instead.